Source/Advance.cpp

```diff
@@ -89,7 +89,7 @@ PeleC::do_mol_advance(
 #endif
 
   // Compute S^{n} = MOLRhs(U^{n})
-  if (verbose) {
+  if (verbose != 0) {
```

Comment:
I looked into doing `verbose` but it looks like we have different levels of verbosity. We have one check for `verbose > 1`.